buildkite/bazelci.py

```diff
@@ -2041,7 +2041,10 @@ def test_label_to_path(tmpdir, label, attempt):
     # remove leading //
     path = label[2:]
     path = path.replace("/", os.sep)
```

Comment:
Slightly simpler solution: replace Line 2042 with path = label.lstrip("/:") and keep the rest